package.json

```diff
@@ -23,6 +23,7 @@
   },
   "homepage": "https://github.com/jsverify/jsverify-contrib#readme",
   "dependencies": {
+    "assert": "^1.3.0",
```

Comment:
When I was reading through the `jsverify` code base, I saw you were using assert. I forgot that it was built into node.